activesupport/lib/active_support/test_case.rb

```diff
@@ -1,14 +1,14 @@
 gem 'minitest' # make sure we get the gem, not stdlib
 require 'minitest'
-require 'active_support/testing/tagged_logging'
-require 'active_support/testing/setup_and_teardown'
+require 'active_support/core_ext/kernel/reporting'
 require 'active_support/testing/assertions'
-require 'active_support/testing/deprecation'
+require 'active_support/testing/constant_lookup'
 require 'active_support/testing/declarative'
+require 'active_support/testing/deprecation'
 require 'active_support/testing/isolation'
-require 'active_support/testing/constant_lookup'
+require 'active_support/testing/setup_and_teardown'
+require 'active_support/testing/tagged_logging'
 require 'active_support/testing/time_helpers'
-require 'active_support/core_ext/kernel/reporting'
```

Comment:
I guess it's unnecessary to change the requires order now that you are not touching it anymore, could you revert that please?